src/Components/WebAssembly/WebAssembly.Authentication/src/ILLink.Descriptors.xml

```diff
@@ -0,0 +1,5 @@
+<linker>
+  <assembly fullname="Microsoft.AspNetCore.Components.WebAssembly.Authentication">
```

Comment:
I understand that this is a safety measure, as it will not trim this assembly at all, but it will also potentially make the app size bigger even after trimming. Unless, of course, if nothing would be trimmed anyway, if trimming was working correctly.

Reply:
No, the assembly is still trimmed, only this type is preserved.

Reply:
The line you reference is only setting up context. Note that `preserve="all"` is on the type element. For other types, the linker has the expected behavior